Assets/Src/Scripts/Animator/AvatarAnimator.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Mediapipe;
+using Mediapipe.Unity;
+using UnityEngine;
+
+namespace SeedUnityVRKit {
+  // <summary>An animator to visualize upper body and face.</summary>
+  public class AvatarAnimator : MonoBehaviour {
```

Comment:
This class is forking UpperBodyAnimator. Please be minded future updates to SeedUnityVRKit will likely break this class's logic.

A better option is to contribute to upstream code, and make any customization via exposing fields to customize the behavior. This guarantees the functionality will continue to be tested when new code is added upstream.